source/common/stream_info/upstream_address_set.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "envoy/network/address.h"
+#include "envoy/stream_info/filter_state.h"
+
+#include "absl/container/flat_hash_set.h"
+
+namespace Envoy {
+namespace StreamInfo {
+
+struct UpstreamAddressSet : public FilterState::Object {
```

Comment:
class comment?
It should be obvious this is the set of upstream addresses, but I read it as "is the upstream address set" because English is ambiguous that way :-P